src/common/snippets/src/pass/validate.cpp

```diff
@@ -82,10 +84,21 @@ bool Validate::is_supported_fq([[maybe_unused]] const std::shared_ptr<const ov::
 }
 
 bool Validate::is_supported_transpose(const std::shared_ptr<const ov::Node>& node) {
-    // Transpose is supported only on Inputs or Outputs of body
+    // Transpose is supported:
+    //  - on Inputs or Outputs of body;
+    //  - or if it is going to be handled by dedicated data flow passes.
```

Comment:
Even if the Transposes will be handled by the dedicated passes, they must still be placed right after Parameter or right before results. Could you please double check why do we need this change?